pag-inicial.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./src/css/style.css">
+    <title>Página Inicial</title>
+</head>
+
+<body>
+
+    <header>
+        
+    </header>
+
+
+    <section class="content-pag-inicial">
+        <div class="textos-pag-inicial">
+            <h1>CONHEÇA NOSSA CAIXINHA!</h1>
+            
+            <p>Um pedacinho do universo geek para a sua casa, todo mês dentro de uma caixa cheia de surpresas!</p>
+            
+            <button> ASSINE JÁ!</button>
+        </div>
+        <div>
+            <a><img class="produtos-pag-inicial" src="./src/img/imagens-pag-inicial/produtos-pag-inicial.png" alt="camiseta, caneca, funko e copo"><a>
+        </div>
+
+    </section>
+
+    <section>
+        <div class="container-logos">
+
+            <a><img class="logos-pag-inicial" src="./images/hp-logo.png" alt="logo harry potter"><a>
```

Comment:
Essa tag de link talvez não precisa estar englobando essas imagens se ela não leva para lugar algum, se for o caso tirar essa tag.